oml/metrics/embeddings.py

```diff
@@ -382,5 +391,9 @@ class EmbeddingMetricsDDP(EmbeddingMetrics, IMetricDDP):
     def sync(self) -> None:
         self.acc = self.acc.sync()
 
+    def update_data(self, data_dict: Dict[str, Any], indices: List[int]) -> None:  # type: ignore
+        # indices are obligatory in DDP, so we don't accumulate shuffled data
```

Comment:
Insufficient comment. The problem is not in shuffling, but in duplicates (at least shuffling is ok in a single device for some reason)

Reply:
okay, changed the comment